xrpVendordep/src/main/java/edu/wpi/first/wpilibj/xrp/XRPGyro.java

```diff
@@ -134,6 +135,15 @@ public double getAngle() {
     return getAngleZ();
   }
 
+  /**
+   * Gets the angle the robot is facing.
+   *
+   * @return A Rotation2d with the current heading.
+   */
+  public Rotation2d getRotation2d() {
+    return new Rotation2d(getAngle());
```

Comment:
@zhiquanyeo Is getAngle CCW+ on the XRP?

```suggestion
    return Rotation2d.fromDegrees(getAngle());
```